src/main/java/com/prgrms/kream/domain/order/dto/request/OrderCreateFacadeRequest.java

```diff
@@ -6,8 +6,8 @@
 public record OrderCreateFacadeRequest(
```

Comment:
```suggestion
public record OrderCreateRequest
```
FacadeRequest지만 Controller에서도 쓰이고 있네요!
네이밍을 변경하는건 어떨까요?

Reply:
컨트롤러에서 파사드로 넘어가는 DTO라.... 이름이 늘 고민이네요.. 고민해보도록하겠습니다!

Reply:
Controller에서 받은 DTO를 Facade로 내릴 때 Mapper를 통해서 내리고 있는데,
생각보다 DTO도 너무 많아지고 Mapper 클래스 메서드도 너무 많아지더라구요 ... 참... 아직 서비스가 얇아서 그렇겠죠 ??
같이 고민해보면 좋을 것 같습니다 !

Reply:
이 부분은 저도 다같이 얘기 나눠보고 싶습니다.
레이어별로 DTO가 나뉘면 그만큼 깔끔한 연관관계라 생각 하는데, 가독성이 많이 저하된다 생각합니다.
뭐가 더 중요할까요!!!